.github/workflows/checks.yml

```diff
@@ -47,6 +47,12 @@ jobs:
         with:
           starknet-foundry-version: latest
 
+      - name: "Create .tool-versions file"
+        run: echo "starknet-foundry 0.15.0" >> .tool-versions
+      - name: "Setup foundry using `.tool-versions` file"
+        uses: ./
+      - run: snforge --version | grep "snforge 0.15.0"
+
```

Comment:
Let's move it to a separate job altogether (something like `test-action-with-tools-file` ?)
Main reason is that at the moment this kind of overshadows `Test local action` step, meaning the whole check will run for 0.15.0 only (until someone updates it)

Having separate job would mean test-action can still test actions against newest versions with some additional things like running snforge test, and the new job could focus only on our new functionality (setting specific snfoundry version using a file, checking if this succeeded).

Reply:
Okay created a seperate job for the action :) lmk if thats better